app/src/main/java/com/x8bit/bitwarden/data/platform/repository/SettingsRepositoryImpl.kt

```diff
@@ -394,6 +401,33 @@ class SettingsRepositoryImpl(
                 initialValue = 0,
             )
 
+    @OptIn(ExperimentalCoroutinesApi::class)
+    override val allVaultSettingsBadgeCountFlow: StateFlow<Int>
+        get() = authDiskSource
+            .activeUserIdChangesFlow
+            .filterNotNull()
+            .flatMapLatest {
+                combine(
+                    vaultDiskSource.getCiphers(it),
+                    authDiskSource.getShowImportLoginsFlow(it),
+                    featureFlagManager.getFeatureFlagFlow(FlagKey.ImportLoginsFlow),
+                ) { ciphers, showImportLogins, importLoginsEnabled ->
+                    val shouldShowImportLogins =
+                        (showImportLogins ?: true) &&
+                            importLoginsEnabled &&
+                            ciphers.isEmpty()
+                    listOf(shouldShowImportLogins)
```

Comment:
So this is intended as a list so you can add more later, right?

I think this is a the same thing you did for the others, right?

Reply:
yep, same idea